src/pages/driver/luggage-policies/index.tsx

```
@@ -140,76 +139,54 @@ const LuggagePolicyRow = ({
+        toast.error(`Failed to delete policy: ${error.message}`);
+      }
     }
   };
 
-  const handleSetDefault = (policyId: string) => {
-    setDefaultMutation.mutate(policyId);
+  const handleSetDefault = async (policyId: string) => {
+    try {
+      await setDefaultMutation({ policyId });
+      toast.success('Default policy updated successfully');
+    } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)